### PR TITLE
Initial starter project set up

### PR DESCRIPTION
**Pull request by sarahmcmorgan**
## Describe your changes

This PR adds the dbt starter project for the Tuva Claims Demo.

* Sample demo data included as seed files.
* Source.yml and logic added to run the project with full demo data. NOTE: This is mentioned in the README. The steps will be added once we get data shares set up.
* Instructions added to the README

## How has this been tested?

This has been tested in Snowflake and Redshift with both sample data and full data.

## Reviewer focus

Please help check for clarity and typos in the README. You can also pull this and try running it yourself to see how it works and ensure the instructions are complete.

## Checklist before requesting a review

- [x]  I have recorded a Loom performing a self-review of my code
- [x]  My code follows style guidelines
- [x]  I have commented my code as necessary
- [x]  I have tested my code by running `dbt build` in the following warehouse(s):
    - [x]  Snowflake
    - [x]  Redshift
- [ ]  I have implemented generic dbt tests to validate primary keys/uniqueness in my model
- [x]  I have updated dbt docs by running `dbt docs generate` and copying the appropriate files to the `docs/` path
- [x]  I have added at least one Github label to this PR

## Loom link
https://www.loom.com/share/d18ee23b95f24556811cb0cd94d18266

## For reviewers

[Best practices for reviewing pull requests.](https://www.notion.so/Pull-Request-Review-Best-Practices-b486354d76c04982889f2178fe777c1c)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,51 @@
...
+# Tuva Claims Demo
+
+## 🧰 What does this project do?
+
+The Tuva Claims Demo provides a quick and easy way to run the Tuva Project with synthetic demo data. 
+You can run the project with built-in sample data loaded as seeds. 
+(A future release will provide the option to download and install the full demo data from data shares.)
+
+For a detailed overview of what the Tuva Project does and how it works, check out our [Knowledge Base](https://thetuvaproject.com/docs/getting-started). 
+To setup the Tuva Project with your own claims demo, please review the ReadMe in [The Tuva Project](https://github.com/tuva-health/the_tuva_project) package for a detailed walkthrough and setup.
```
Comment (aneiderhiser) [inline README.md]: should this say: "claims data" instead of "claims demo"?

File: README.md [hunk 2]
```diff
@@ -1 +1,51 @@
...
+
+For information on data models and to view the entire DAG check out our dbt [Docs](https://tuva-health.github.io/tuva_claims_demo/#!/overview?g_v=1).
```
Comment (aneiderhiser) [inline README.md]: This gives me an error when I try to go to the docs link.

**Review by aneiderhiser — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sarahmcmorgan) [reply README.md]: @aneiderhiser Yep! Good catch. 🙃

File: README.md (hunk 2 above)
Comment (sarahmcmorgan) [reply README.md]: @aneiderhiser That's expected. The page has not been built yet. It will be built once this initial PR goes in.

**Review by aneiderhiser — APPROVED** [review]

File: dbt_project.yml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  # dumped into. Make sure this database exists
+  # in your data warehouse before you run this package,
+  # since dbt can create schemas in your data warehouse,
+  # but it cannot create databases.
+  tuva_database: tuva_claims_demo_sample
+
+  # This variable allows you to override the sample data seeds and
+  # use the full demo data instead, you may also want to update the
+  # tuva_database variable to not overwrite the sample demo
+  full_data_override: false
```
Comment (sarahmcmorgan) [inline dbt_project.yml:43]: @aneiderhiser I improved this logic a bit. It's now a separate var and not tied to the database name. I also updated the steps in the README to reflect this change.

Comment (forrestbeckwith) [reply dbt_project.yml:43]: awesome, was going to say after using it I was going to suggest something like this instead.

**Comment by forrestbeckwith** [comment]
Ran through the project, looks great!

My only question, which I understand there isn't a great solution for, is are the seeds (specifically the medical_claim seed) still too big?  Even with just 100 patients, loading the medical_claim seed took me ~25 minutes (and even longer the second time because my computer went to sleep while I got up in the middle).  I think that long of a load time might be a hinderance to people using the demo; people might think its not working and turn it off part way through, or might get the sense that the tuva project itself is slow, even if the actual tuva project code runs in a couple minutes.

It seems like we're close on the public data set?  My initial thought is maybe we really push on that for demos where we want to show the quality of the output and the kind of value it can have, and maybe the seed version is smaller and really just have it there as a way to show how the mechanics of the tuva project work and what the table structure etc will look like?  I wonder if we had like 10 patients, or if we kept the 100 but just unceremoniously dropped a bunch of the medical_claim records if we could get the load time for that seed to ~5 minutes?  Maybe just the wideness of the table might make that difficult to do even at a low record count :/

**Review by forrestbeckwith — APPROVED** [review]
Looks good!  I still have some concerns about the load time for the medical claim seed, but I don't think that should hold up the release, great job!

**Pull request merged**